src/docs/asciidoc/reference/configuration/configuration-properties.adoc

```diff
@@ -1,6 +1,26 @@
 :relative-path: ../../
 include::{docdir}/variables.adoc[]
 
+ZetaPush provides the `application.json` file to let you configure the behavior of your application.
 
+For now, the `application.json` file is mainly used for the <<reference#integrate_user_management, Standard User Workflow>> but you can also <<reference#configuration_properties ,configure your own properties for your application>>.
+The list of available properties for the <<reference#integrate_user_management, Standard User Workflow>> are listed in <<reference#customize_standard_user_workflow, this section>>.
+
+.New properties will added
+[NOTE]
+====
+New _Cloud Services_ will be developed (to handle data, security, etc...).
+With them, some available properties of the `application.json` will be added.
+====
+
+.Futur features
+[NOTE]
+====
+New features about `application.json` file will be soon released :
+
+* Support of multi `application.json` file, one per worker (when the multi-worker) will be released.
+* Support of `application.json` file per environment like `application-<env>.json` for example. (When the multi-environment will be released)
+* Autocompletion of properties in VSCode (and other IDE) 
+====
```

Comment:
We should also add information about merging of properties and priority order (from less priorized to most priorized):

1) application.json for whole application (available in future version)
2) application-<env> shared for whole application (available in future version)
3) application.json shared between all workers (available in future version)
4) application-<env> shared between all workers (available in future version)
5) application.json for a worker
6) application-<env>.json for a worker (available in future version)
7) Environment variable (provided either by terminal or Web Console)
8) CLI argument

I saw the priority order after. Maybe here we should just link to the explanation but the explanation must be updated to provide more information.

Reply:
I think we don't need to have a lot of information about priority order for now.
This will be more useful in the future versions